Sources/FirebaseRemoteCommand.swift

```diff
@@ -80,16 +80,19 @@ public class FirebaseRemoteCommand: RemoteCommand {
                     firebaseInstance.logEvent(eventName, items(from: payload))
                     return
                 }
-                if let items = params[FirebaseConstants.Keys.paramItems] as? [[String: Any]] {
+                normalizedParams += mapParams(params)
+                if let itemsArray = params[FirebaseConstants.Keys.paramItems] as? [[String: Any]] {
                     var tempItems = [[String: Any]]()
                     var item = [String: Any]()
-                    items.forEach {
-                        item = eventParameters.map($0)
+                    itemsArray.forEach {
+                        item = mapParams($0)
```

Comment:
could simplify this to `let item = mapParams($0)` without needing to declare the var outside the loop